### ISSUE TITLE
Screen blinks(redrawing i guess) after #247

### ISSUE THREAD

**Issue by monkoose** [Shougo/deoplete.nvim#252]
# Problems summary
After fix of the #247 issue, deoplete makes screen blinks(redrawing i guess because my vb is disabled) when it can't find completion for current word if there is 2 buffers or split windows. 
Actually i can't reproduce this issue with my minimal init.vim, because it seems like it still doesn't complete from other buffers with minimal init.vim(some option should be enabled?). But with my actual init.vim completions from other buffers work but with screenblinks after each letter.

## Environment Information

 * OS: Arch Linux
 * Neovim version: 0.1.3

## Provide a minimal init.vim with less than 50 lines (Required!)

```vim
" Your minimal init.vim
set runtimepath+=~/path/to/deoplete.nvim/
let g:deoplete#enable_at_startup = 1

```

Do you need more info(logs or something?), let me know.

**Comment by Shougo** [comment]
OK, it is reproduced.
To make the other buffer cache is always flicks.
Unfortunately, the cache feature is disabled.

**Comment by monkoose** [comment]
Thanks.
Its fine for me for now, because blinks are annoying, and competions still work if you previously edit another buffer. 
But how then your neocomplete works fine without flicks?

**Comment by Shougo** [comment]
I think it is neovim python's feature...
It is annoying though.

**Comment by Shougo** [comment]
I have re-implemented #247.

**Comment by monkoose** [comment]
Thank you, will try it.
